libs/langchain-community/src/vectorstores/cassandra.ts

```diff
@@ -1,8 +1,5 @@
 /* eslint-disable prefer-template */
-import {
-  AsyncCaller,
-  AsyncCallerParams,
-} from "@langchain/core/utils/async_caller";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Big diff but I assume this is mostly just moving code around?

Reply:
Yes, the table-related logic moved to utils and consolidated into the `CassandraTable` class, so `vectorstore` was most impacted; `message` then got updated to make use of the `CassandraTable` class rather than accessing the client directly.